webapp/portlet/src/main/webapp/skin/less/common/siteDetails/Style.less

```diff
@@ -0,0 +1,52 @@
+/**
+
+  This file is part of the Meeds project (https://meeds.io/).
+
+  Copyright (C) 2020 - 2023 Meeds Association contact@meeds.io
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+*/
+
+@import "../../variables.less";
+@import "../../mixins.less";
+
+.siteDetails {
+  #siteNavigationTree {
+    .v-treeview-node__content {
+      margin-left: 0px  ~'!important; /** orientation=lt */ ';
+      margin-right: 0px ~'!important; /** orientation=rt */ ';
+    }
+    .v-list-item--active {
+      color: @textLightColor!important;
+      background: @greyColorLighten1Opacity1;
+      &:before {
+        content: '';
+        width: 8px;
+        background:@secondaryColor;
+        position: absolute;
+        left: 0;
+        opacity: 1!important;
+      }
+      .v-list-item__icon {
+        i {
+          color: @toolbarLightIconColor !important;
+        }
+        .spaceRightArrow {
+          color: @toolbarDarkLinkHoverColor !important;
+        }
+      }
+    }
+  }
```

Comment:
These classes are too specific for certain elements and do not exist in helpers or Vuetify classes.
it is similar to hamburger menu menu classes with some changes

Reply:
As [mentioned here](https://github.com/Meeds-io/social/pull/3229#discussion_r1404536475), we will keep this for now to not block the merge. But please, we doesn't need CSS classes anymore since we use Material Design classes, Vuetify classes and specific CSS Class helpers.